### PR TITLE
WIP-inspector: refactoring test and removing flag

### PR DESCRIPTION
**Pull request by kimj42**
**What** **and **where** it changed:**
   - Removed 'react-inspector' flag in DebugConsoleTest, DebugConsole, Redux, jsDebugger, App Lab, and Game Lab. Added test for when flag is removed in Redux Test. Removed tests that existed for when flag was off in DebugConsoleTest and ReduxTest.

**Why** **are we making this change**?
   - Although there are additional changes to be made such as making the error output display as text instead of strings by going through the inspector tag, we are removing the flag now because there is a lot that students can do with the current state of the flag: able to expand objects/lists and colorized output from both console.logging from code workspace and also from debug console. 

**How you tested the change**:
   - Wrote unit tests for checking that input and output from debug console is displayed correctly in the debug console. Also wrote unit test for checking logOutput state for redux. 

**Result of removing inspector flag**:
Screen Shot 2019-06-18 at 9 32 31 AM

### CODE REVIEW

File: apps/src/lib/tools/jsdebugger/DebugConsole.jsx [hunk 1]
```diff
@@ -160,33 +157,29 @@ export default connect(
...
-              );
-              this.appendLog({
-                output: result,
-                undefinedInput: input === 'undefined' ? true : false
-              });
-            } else {
-              const result = this.props.evalInCurrentScope(input);
-              this.appendLog('< ' + String(result));
-            }
+            // if (this.props.showReactInspector) {
```
Comment (epeach) [inline apps/src/lib/tools/jsdebugger/DebugConsole.jsx]: Are these leftover comments?

**Review by epeach — COMMENTED** [review]
Looking good! Just a little clean-up work.

File: apps/src/lib/tools/jsdebugger/DebugConsole.jsx (hunk 1 above)
Comment (kimj42) [reply apps/src/lib/tools/jsdebugger/DebugConsole.jsx]: Woops! Yes, I'll go ahead and remove those.

**Comment by kimj42** [comment]
Created updated version of this WIP PR so will close this PR

**Pull request closed without merging**